pages/vi/treehouses-remote.md

```diff
@@ -36,5 +36,5 @@ The app may ask for your device location, feel free to decline.
 
 **NOTE:** The time it takes to reboot is about 2-4 minutes.
 
-**Note for later:** You will need to unpair and re-pair eveytime you use a new Image version
+**Note for later:** You will need to unpair, re-pair, and restart your phone evey time you use a new image version.
```

Comment:
A note should be in the blue box. I think this one should be a warning instead of a note.